settings.ini

```diff
@@ -8,14 +8,17 @@ author = Yiwei Mao, et al.
 author_email = yiwei.mao@sydney.edu.au
 copyright = Yiwei Mao, et al.
 branch = master
-version = 0.2.1
+version = 0.2.2
 min_python = 3.7
 audience = Developers
 language = English
 custom_sidebar = True
 license = apache2
 status = 4
-requirements = numpy scipy matplotlib nbdev fastcore fastprogress h5py xarray pillow pandas tqdm bokeh holoviews panel param Py6S pyserial netcdf4 streamz pyviz_comms hvplot requests<=2.24.0 datashader psutil
+requirements = numpy scipy matplotlib fastcore fastprogress h5py xarray pillow pandas tqdm bokeh holoviews panel param Py6S pyserial netcdf4 streamz pyviz_comms hvplot requests<=2.24.0 datashader psutil
+pip_requirements = opencv-python
+conda_requirements = 
+dev_requirements = nbdev conda-build twine
```

Comment:
I think we should remove opencv-python from pip_requirements. It was really only for testing with webcams - not used in `SimulatedCamera`. This format separating the pip and conda makes sense though and I like it.

Reply:
done.